tests/ats/test_basic_cluster.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Dict, List
+
+import pykube
+import pytest
+from pytest_helm_charts.clusters import Cluster
+from pytest_helm_charts.k8s.deployment import wait_for_deployments_to_run
+
+logger = logging.getLogger(__name__)
+
+namespace_name = "default"
+
+timeout: int = 360
+
+
+@pytest.mark.smoke
+def test_api_working(kube_cluster: Cluster) -> None:
+    """Very minimalistic example of using the [kube_cluster](pytest_helm_charts.fixtures.kube_cluster)
+    fixture to get an instance of [Cluster](pytest_helm_charts.clusters.Cluster) under test
+    and access its [kube_client](pytest_helm_charts.clusters.Cluster.kube_client) property
+    to get access to Kubernetes API of cluster under test.
+    Please refer to [pykube](https://pykube.readthedocs.io/en/latest/api/pykube.html) to get docs
+    for [HTTPClient](https://pykube.readthedocs.io/en/latest/api/pykube.html#pykube.http.HTTPClient).
+    """
+    assert kube_cluster.kube_client is not None
+    assert len(pykube.Node.objects(kube_cluster.kube_client)) >= 1
+
+
+@pytest.mark.smoke
+def test_cluster_info(
+    kube_cluster: Cluster, cluster_type: str, chart_extra_info: Dict[str, str]
+) -> None:
+    """Example shows how you can access additional information about the cluster the tests are running on"""
+    logger.info(f"Running on cluster type {cluster_type}")
+    key = "external_cluster_type"
+    if key in chart_extra_info:
+        logger.info(f"{key} is {chart_extra_info[key]}")
+    assert kube_cluster.kube_client is not None
+    assert cluster_type != ""
+
+
+# scope "module" means this is run only once, for the first test case requesting! It might be tricky
+# if you want to assert this multiple times
+@pytest.fixture(scope="module")
+def app_deployment(kube_cluster: Cluster) -> List[pykube.Deployment]:
+    deployments = wait_for_deployments_to_run(
+        kube_cluster.kube_client,
+        ["trivy-operator"],
```

Comment:
So this will just check for the deployment of `trivy-operator`, we could even add a test for all pack components, WDYT?